.travis.yml

```diff
@@ -5,10 +5,10 @@ branches:
 language: node_js
 node_js:
   - "8"
-script:
-  - bash ./src/scripts/deploy.sh
 sudo: required
 install:
   - npm install --global yarn
   - yarn
   - sudo pip install proselint
+script:
```

Comment:
Why this change?

Reply:
`install` happens before the `script`, so it makes more sense to move `script` at the end, where it actually takes place. Doesn't really affect functionality, just to make config more clear to read